syntax="proto2";

package aap_protobuf.service.navigationstatus;


enum NavigationStatusMessageId
{
  INSTRUMENT_CLUSTER_START = 32769;
  INSTRUMENT_CLUSTER_STOP = 32770;
  INSTRUMENT_CLUSTER_NAVIGATION_STATUS = 32771;
  INSTRUMENT_CLUSTER_NAVIGATION_TURN_EVENT = 32772 [deprecated = true];
  INSTRUMENT_CLUSTER_NAVIGATION_DISTANCE_EVENT = 32773 [deprecated = true];
  INSTRUMENT_CLUSTER_NAVIGATION_STATE = 32774;
  INSTRUMENT_CLUSTER_NAVIGATION_CURRENT_POSITION = 32775;
}
